### ISSUE TITLE
App crashing on Big Sur 11.4

### ISSUE THREAD

**Issue by Human79** [AvaQueen-Official/About-This-Hack#10]
App crashing on Big Sur 11.4.
![Screen Shot 2021-07-01 at 12 07 49 PM](https://user-images.githubusercontent.com/84245065/124077296-fc55d380-da64-11eb-9620-24aeadf5523b.png)

**Comment by 8itCat** [comment]
@Human7900 what mac is this and can you sent the crash log?

**Comment by Human79** [comment]
The SMBIOS is MacBookPro13,1. How can I send you the crash log?

**Comment by 8itCat** [comment]
Click "Report" then there should be an option to save a text file. If not, a screenshot will do

**Comment by i-do-know-who-i-am** [comment]
Here.
[Log.txt](https://github.com/AvaQueen-Official/About-This-Hack/files/6749332/Log.txt)

**Comment by Human79** [comment]
Here.
[Log.txt](https://github.com/AvaQueen-Official/About-This-Hack/files/6749358/Log.txt)

**Comment by 8itCat** [comment]
@Human7900 do you mind DMing me on Discord via my server? Crash logs aren't helping much - it seems every framework is crashing for no apparent reason.

Link:
https://dsc.gg/hackcaf%C3%A9
My nickname is `ASentientAva`

Thanks in advance!

**Comment by Human79** [comment]
The invite is expired. Can send it again?

**Comment by 8itCat** [comment]
sorry about that - https://discord.gg/5AQjAnNKYd

**Comment by Kief5555** [comment]
Same Issue here, I got a crash

**Comment by Kief5555** [comment]
(And yes, You may remember me) Kief On discord

**Comment by 8itCat** [comment]
do you guys get something like this when you run About This Hack > Show Package Contents > Contents > MacOS > About This Hack in terminal?
~ % /Applications/About\ This\ Hack.app/Contents/MacOS/About\ This\ Hack ; exit;
macOS 11.5.2 (20G95)
11.5.2
C02XXXXXXXXX
Serial Number: C02XXXXXXXXX
Print: Entry, ":CPU Names:Q6LR-en-US_US", Does Not Exist
16
16 GB
zsh: trace trap  /Applications/About\ This\ Hack.app/Contents/MacOS/About\ This\ Hack
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[Process completed]

**Comment by mehmetyukcell** [comment]
is there any progress?

**Comment by 0xCUB3** [comment]
> is there any progress?

@Human79 @Kief5555 @mehmetyukcell yep should be fixed in my new repo: https://github.com/0xCUB3/About-This-Hack
